### PR TITLE
Update frontend dependencies

### PR DESCRIPTION
**Pull request by trustin**
- Lots of dependency updates as usual. Please see the diff.
- Added `TERM=dumb` environment variable when launching Gatsby to work around https://github.com/gatsbyjs/gatsby/issues/28723

### CODE REVIEW

File: site/gatsby-config.js [hunk 1]
```diff
@@ -14,6 +14,13 @@ module.exports = {
       },
     ],
   },
+  flags: {
+    DEV_SSR: true,
+    FAST_DEV: true,
+    FAST_REFRESH: true,
```
Comment (ikhoon) [inline site/gatsby-config.js:20]: 👍

**Review by ikhoon — APPROVED** [review]
Thanks! @trustin 🙇‍♂️

File: site/gatsby-config.js (hunk 1 above)
Comment (trustin) [reply site/gatsby-config.js:20]: Doesn't seem to make anything faster though.

Comment (trustin) [reply site/gatsby-config.js]: Reverted :laughing:

**Review by minwoox — APPROVED** [review]
Thanks!

**Pull request merged**